source/include/FreeRTOS_IPv6_Private.h

```diff
@@ -55,22 +55,16 @@
 #undef TCP_PACKET_SIZE
 #define TCP_PACKET_SIZE          ( sizeof( TCPPacket_IPv6_t ) )
 
-/* The offset from the UDP payload where the IP type will be stored.
- * For IPv4 packets, this it located 6 bytes before pucEthernetBuffer.
- * For IPv6 packets, this it located in the usual 'ucVersionTrafficClass'. */
-#define ipIP_TYPE_OFFSET         ( 6U )
 /* The offset into an IP packet into which the IP data (payload) starts. */
 #define ipIPv6_PAYLOAD_OFFSET    ( sizeof( IPPacket_IPv6_t ) )
```

Comment:
I don't see any reference to `ipIPv6_PAYLOAD_OFFSET` in the dev/IPv6_integration branch, might as well remove it.
In stead `ipUDP_PAYLOAD_OFFSET_IPv6` is used everywhere.